fenix/app/src/main/res/values/strings.xml

```diff
@@ -1283,7 +1283,10 @@
 
     <!-- Text shown in the notification that pops up to remind the user that a private browsing session is active for Android 14+ -->
     <string name="notification_erase_title_android_14">Close private tabs?</string>
-    <string name="notification_erase_text_android_14">Tap or swipe this notification to close private tabs.</string>
+    <!-- Text shown in the notification description that pops up to remind the user that a private browsing session is active
+    for Android 14+. "Clear" is used in Android to denote the notification being dismissed, typically, this is by swiping the
+    notification away.-->
+    <string name="notification_erase_text_android_14">Tap or clear this notification to close private tabs.</string>
```

Comment:
The cut has happened so I think we need to append this as a new `_2` string and go through the normal moz:removedIn process. We can check with l10n team (Delphine) to see if this could be avoided, but that depends on whether or not the strings have already been imported and exposed for localization

Reply:
@gabrielluong Thank you for checking this. We are already aware of the issue, we are waiting for a confirmation from UX regarding the string that neeeds to be changed ( Fenix or Focus)